submissions/2024-10-11-MLPepper-RECAP-Optimized-Fragments-Add-Iodines-v1.0/README.md

```diff
@@ -0,0 +1,93 @@
+# MLPepper-RECAP-Optimized-Fragments-Add-Iodines-v1.0
+
+## Description
+
+A single point dataset created by combining the [50k ESP from Simon](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2022-01-16-OpenFF-ESP-Fragment-Conformers-v1.0) and 
+[Br substituted set from Lily](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2023-11-30-OpenFF-multi-Br-ESP-Fragment-Conformers-v1.1-single-point), filtering by Cl and Br and replacing them successively with iodines, as well as some additional iodines from [Lexi and Lily's set](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0).
+Each fragment had 5 conformations generated which were optimised locally using an AIMNET2 model trained to `wb97m-d3`. 
+This adds an extension of iodines molecules to the [original mlpepper dataset](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-07-26-MLPepper-RECAP-Optimized-Fragments-v1.0).
+
+The aim of the dataset is to provide polarised and gas phase electrostatic properties which can be used to generate ML models 
+for partial charge prediction. Unlike past datasets the wavefunction will not be saved to recompute the ESP instead we recommend building the ESP 
+from the MBIS atomic multipoles which save substantial amount of space. 
+
+An off equilibrium data set will also be generated to enable conformation dependent prediction of charges. 
+
+## General Information
+
+
+* Date: 2024-10-11
+* Class: OpenFF SinglePoint Dataset
+* Purpose: Electrostatic properties for ML prediction models 
+* Name: MLPepper-RECAP-Optimized-Fragments-Add-Iodines-v1.0
```

Comment:
The name here should be the name used to extract the dataset from qcarchive and should match the `dataset.dataset_name` field.